### ISSUE TITLE
LF1 Check It

### ISSUE THREAD

**Issue by tdegeorge** [TeamBasedInquiryLearning/precalculus#353]
Reviewer: @AbbyANoble 

Task 1: (two options)

Option 1: Something like: Find the average rate of change of f(x) = 0.001 x^3 + 0.05x^2 - 0.25x + 600 on the interval [200, 300]. 

Option 2: Give a story and ask to calculate the average rate of change. For example:

I just put out a new rain barrel to catch the rain water that falls on my roof. It started raining at 10:45 am and by 11:40 am, the barrel had 6 inches of water in it. 
a) What was the average rate of change of the height of the water during this time period? Include units in your answer.
b) If the rain kept falling at this same constant rate, when would the rain height in the barrel reach 12 inches?

Task 2: Give students two points with a value missing in one of the points and the slope to find the missing value.
Example: Find the value of t such that the slope of the line that passes through (-5, 6) and (t, -12) is -3/4.

**Comment by AbbyANoble** [comment]
Task 1:
I like both options. I think two tasks on average rate of change could make sense. 

For option 1, would it always be a cubic with lots of decimals? You don't have any in the section like that. And sometimes the ones that have a lot to plug into lead to a typo even when they understood the concept. I also think it would make sense also to vary the function. Sometimes a polynomial, square root, rational, etc. 

For option 2, the only issue I see is that multiple contexts will have to be written. I'm not sure how many different prompts we need to make the generated problems feel different enough. Maybe @siwelwerd and/or @StevenClontz can speak to that.

**Comment by AbbyANoble** [comment]
Task 2: 
The objective of the section states to find the slope of the line, so if we only have one question on slope I think it should be a straightforward one asking them to find a slope. Maybe with a few parts, like one given a graph, one given an equation, and one given two points.

This question could be a good second task on slope since it's not asking them to do the primary objective. I'm also not seeing anything like this in the lesson. Could be an activity there too!

**Comment by tdegeorge** [comment]
> Task 1: I like both options. I think two tasks on average rate of change could make sense.
> 
> For option 1, would it always be a cubic with lots of decimals? You don't have any in the section like that. And sometimes the ones that have a lot to plug into lead to a typo even when they understood the concept. I also think it would make sense also to vary the function. Sometimes a polynomial, square root, rational, etc.
> 
> For option 2, the only issue I see is that multiple contexts will have to be written. I'm not sure how many different prompts we need to make the generated problems feel different enough. Maybe @siwelwerd and/or @StevenClontz can speak to that.

Makes sense. I feel like it might be hard to generate lots of different types of problems. I took out the word problem one and left option 1 for the first task. I like the idea of varying the types of equations students are given.

**Comment by tdegeorge** [comment]
This was the original Option 2 I had in case Drew or Steven come looking here :)

Option 2: Give a story and ask to calculate the average rate of change. For example:

I just put out a new rain barrel to catch the rainwater that falls on my roof. It started raining at 10:45 am and by 11:40 am, the barrel had 6 inches of water in it. 
a) What was the average rate of change in the height of the water during this period? Include units in your answer.
b) If the rain kept falling at this same constant rate, when would the rain height in the barrel reach 12 inches?

**Comment by tdegeorge** [comment]
> Task 2: The objective of the section states to find the slope of a given line, so if we only have one question on slope I think it should be a straightforward one asking them to find a slope. Maybe with a few parts, like one given a graph, one given an equation, and one given two points.
> 
> This question could be a good second task on slope since it's not asking them to do the primary objective. I'm also not seeing anything like this in the lesson. Could be an activity there too!

So basically like Activity [[Activity 3.1.21](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF1.html#calculate-slope-using-formula) 
https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF1.html#calculate-slope-using-formula](url)?

**Comment by StevenClontz** [comment]
> For option 2, the only issue I see is that multiple contexts will have to be written. I'm not sure how many different prompts we need to make the generated problems feel different enough. Maybe @siwelwerd and/or @StevenClontz can speak to that.

My instinct is to make part 2 always have an example of each kind: "the line passing through (point) and (point)", "the line with xy values given by this chart", "the line shown in the below graph".
